easybuild/tools/github.py

```diff
@@ -395,6 +407,235 @@ def post_comment_in_issue(issue, txt, repo=GITHUB_EASYCONFIGS_REPO, github_user=
         raise EasyBuildError("Failed to create comment in PR %s#%d; status %s, data: %s", repo, issue, status, data)
 
 
+def _copy_easyconfigs_to_repo(paths, target_dir):
+    """
+    Copy files to git working directory to stage/commit & create PR
+
+    @paths: list of paths to copy to git working dir
+    @target_dir: target directory
+    """
+    file_info = {
+        'ecs': [],
+        'paths_in_repo': [],
+        'new': [],
+    }
+
+    a_to_z = [chr(i) for i in range(ord('a'), ord('z') + 1)]
+    subdir = os.path.join('easybuild', 'easyconfigs')
+
+    if os.path.exists(os.path.join(target_dir, subdir)):
+        for path in paths:
+            ecs = process_easyconfig(path)
```

Comment:
You should run into something like `IOError: Failed to read file ...`.

It can check here (or in process_easyconfig) whether it's a file though, doesn't hurt...